manifests/o/Oracle/JavaRuntimeEnvironment/8.0.3330.2/Oracle.JavaRuntimeEnvironment.installer.yaml

```
@@ -0,0 +1,52 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Oracle.JavaRuntimeEnvironment
+PackageVersion: 8.0.3330.2
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+Scope: machine
+InstallModes:
+- interactive
+- silent
+InstallerSwitches:
+  Silent: /s REBOOT=0
+  SilentWithProgress: /s REBOOT=0
+  InstallLocation: INSTALLDIR=<INSTALLPATH>
+  Custom: SPONSORS=0 AUTO_UPDATE=0
+UpgradeBehavior: uninstallPrevious
+Commands:
+- java
+FileExtensions:
+- class
+- idx
+- jad
+- jar
+- java
+- jsp
+Installers:
+- Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://javadl.oracle.com/webapps/download/AutoDL?BundleId=246474_2dee051a5d0647d5be72a7c0abff270e
+  InstallerSha256: B0244A6DD643F7D2B6F891B7BC2BAECA6142897CDDEE877A83F1E7F89EFCB3FA
+  AppsAndFeaturesEntries:
+  - DisplayName: Java 8 Update 331 (64-bit)
+    Publisher: Oracle Corporation
+    DisplayVersion: 8.0.3330.2
+    ProductCode: '{26A24AE4-039D-4CA4-87B4-2F64180331F0}'
+    InstallerType: msi
+- Architecture: x86
+  InstallerType: exe
+  InstallerUrl: https://javadl.oracle.com/webapps/download/AutoDL?BundleId=246472_2dee051a5d0647d5be72a7c0abff270e
+  InstallerSha256: 4CEB40B04E3A564D4C8C9342174A5CBDBA373B8A0C4BE0D5E9756903CE551CE4
+  AppsAndFeaturesEntries:
+  - DisplayName: Java 8 Update 331
+    Publisher: Oracle Corporation
+    DisplayVersion: 8.0.3330.2
+    ProductCode: '{26A24AE4-039D-4CA4-87B4-2F32180331F0}'
+    InstallerType: msi

```

```suggestion
  - DisplayName: Java 8 Update 333
    Publisher: Oracle Corporation
    ProductCode: '{26A24AE4-039D-4CA4-87B4-2F32180333F0}'
    InstallerType: msi
```
Updated DisplayName to match Control Panel.
Removed DisplayVersion since it's not needed because the PackageVersion matches Control Panel.
Fixed ProductCode